code/controllers/subsystem/idlenpcpool.dm

```diff
@@ -39,8 +39,6 @@ SUBSYSTEM_DEF(idlenpcpool)
 			continue
 
 		if(!SA.ckey)
-			if(SA.stat != DEAD)
-				SA.handle_automated_movement()
```

Comment:
I hate wander and apparently when a AI is idle it will still wander... just in the middle of nowhere. Im most likely going to have to put this back. At the top the idleNPC pool has a 6 second delay so i wonder if that is why some AI are blindsided when a player runs past them.